Migrating-MySQL-Database-to-Spark/README.md

```diff
@@ -0,0 +1,135 @@
+# Migrating MySQL Database to Spark
```

Comment:
I would suggest to write a blog post instead of adding this to our documentation?